views/reports/custom_reports_table.php

```diff
@@ -0,0 +1,48 @@
+﻿<?php
```

Comment:
This file is saved using the encoding `UTF-8-BOM` which could have occurred if you were using notepad to write the file. Change this file encoding to `UTF-8` (the `UTF-8-BOM` adds a whitespace to the top of this page)